child_compassion/models/child_compassion.py

```diff
@@ -411,7 +412,15 @@ def get_infos(self):
         """Get the most recent case study, basic informations, updates
            portrait picture and creates the project if it doesn't exist.
         """
-        # TODO Implement with new service
+        message_obj = self.env['gmc.message.pool']
+        action_id = self.env.ref(
+            'child_compassion.beneficiaries_details').id
+
+        message_vals = {
+            'action_id': action_id,
+            'object_id': self.id,

```

Comment:
Please add as well field `child_id : self.id` (for getting to the child from the view)